rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/UnityHost.kt

```diff
@@ -32,7 +32,7 @@ import com.sun.jna.Native
 import com.sun.jna.win32.StdCallLibrary
 import java.awt.Frame
 
-class UnityHost(project: Project, runManager: RunManager) : LifetimedProjectComponent(project) {
```

Comment:
Please use ProtocolSubscribedProjectComponent as a base class. It'll be easier to migrate this class to `protocol.modelListener` when it's ready